framework/src/modules/chain/modules/blocks/process.js

```diff
@@ -537,68 +547,40 @@ Process.prototype.loadBlocksFromPeer = function(peer, cb) {
  * @returns {function} cb - Callback function from params (through setImmediate)
  * @returns {Object} cb.err - Error message if error occurred
  */
-Process.prototype.generateBlock = function(keypair, timestamp, cb) {
+Process.prototype.generateBlock = async function (keypair, timestamp, cb) {
```

Comment:
This function is not an async function. Therefore, we shouldn't make it one. I would suggest that when calling `modules.processTransactions.verifyTransactions`, we use `.then` and `.catch` blocks which call the callback functions.

Reply:
@SargeKhan I think we cannot use `await` in non async function?

Reply:
@SargeKhan why not possible to turn it into an async/await function?

Reply:
after discussion with @shuse2 decision to use then to maintain callback usage, and to do refactoring into async at once in a later PR.